Source/missiles.cpp

```diff
@@ -676,23 +676,17 @@ void __fastcall DeleteMissile(int mi, int i)
 
 void __fastcall GetMissileVel(int i, int sx, int sy, int dx, int dy, int v)
 {
-	int v6;     // eax
-	double v7;  // ST18_8
-	double v8;  // ST10_8
-	int v9;     // esi
-	double v10; // st7
+	double dxp,dyp,dr;
 
 	if (dx != sx || dy != sy) {
-		v7 = (double)((dx + sy - sx - dy) << 21);
-		v8 = (double)((dy + dx - sx - sy) << 21);
-		v9 = i;
-		v10 = 1.0 / sqrt(v8 * v8 + v7 * v7);
-		missile[v9]._mixvel = (signed __int64)((double)(v << 16) * v7 * v10);
-		missile[v9]._miyvel = (signed __int64)((double)(v << 15) * v8 * v10);
+		dxp = (double)((dx + sy - sx - dy) << 21);

```

Comment:
Casting isn't strictly needed as it gets assigned to a double.
```suggestion
		dxp = (dx + sy - sx - dy) << 21;
```
(but I don't know if LLVM or MSVC will like that)